### PR TITLE
Feature/EUI 8901

### PR DESCRIPTION
**Pull request by DavidJayakumar**
### Jira link (if applicable)

https://tools.hmcts.net/jira/browse/EUI-8901

### Change description ###

Add translation to hearings journey

### Checklist

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)
- [x] Does this PR introduce a breaking change

### CODE REVIEW

File: src/hearings/containers/hearing-actuals/hearing-actuals-edit-summary/hearing-actuals-edit-summary.component.html [hunk 1]
```diff
@@ -1,204 +1,217 @@
...
-          <dt class="govuk-summary-list__key">Case name</dt>
-          <dd class="govuk-summary-list__value" [innerHtml]="answerSource.CASE_NAME | transformAnswer:hearingState$ | async"></dd>
+          <dt class="govuk-summary-list__key">{{getHearingDateText()}}</dt>
+          <dd class="govuk-summary-list__value">{{hearingDateRange}}</dd>
         </div>
         <div class="govuk-summary-list__row">
-          <dt class="govuk-summary-list__key">Case number</dt>
-          <dd class="govuk-summary-list__value" [innerHtml]="answerSource.CASE_NUMBER | transformAnswer:hearingState$ | async"></dd>
+          <dt class="govuk-summary-list__key">{{ 'Paper hearing' | rpxTranslate }}</dt>
+          <dd class="govuk-summary-list__value">{{isPaperHearing ? 'Yes' : 'No'}}</dd>
```
Comment (johnbenjamin-hmcts) [inline src/hearings/containers/hearing-actuals/hearing-actuals-edit-summary/hearing-actuals-edit-summary.component.html:36]: Yes No translation required?

File: src/hearings/containers/hearing-actuals/hearing-actuals-edit-summary/hearing-actuals-edit-summary.component.html [hunk 2]
```diff
@@ -1,204 +1,217 @@
...
-                    <dd class="govuk-summary-list__actions">
-                      <a href="javascript:void(0)" class="govuk-link" [routerLink]="hearingActualAddEditUrl()" fragment="hearingDate">Change</a>
-                    </dd>
+            <div class="govuk-details__text listing-info">
+              <dl class="govuk-summary-list govuk-!-margin-bottom-0 govuk-!-padding-right-2">
+                <div class="govuk-summary-list__row">
+                  <dt class="govuk-summary-list__key ">{{ 'Was this hearing day required?' | rpxTranslate }}</dt>
+                  <dd class="govuk-summary-list__value">
+                    <div>
+                      {{actualHearingDay.notRequired ? 'No' : 'Yes'}}
```
Comment (johnbenjamin-hmcts) [inline src/hearings/containers/hearing-actuals/hearing-actuals-edit-summary/hearing-actuals-edit-summary.component.html:91]: Yes No translation required?

File: src/hearings/containers/hearing-actuals/hearing-actuals-edit-summary/hearing-actuals-edit-summary.component.html [hunk 3]
```diff
@@ -1,204 +1,217 @@
...
+                    </div>
+                  </dd>
+                  <dd class="govuk-summary-list__actions">
+                    <a href="javascript:void(0)" class="govuk-link" [routerLink]="hearingActualAddEditUrl()"
+                      fragment="hearingDate">Change</a>
```
Comment (johnbenjamin-hmcts) [inline src/hearings/containers/hearing-actuals/hearing-actuals-edit-summary/hearing-actuals-edit-summary.component.html:96]: Translation for Change link.

File: src/hearings/containers/hearing-actuals/hearing-actuals-edit-summary/hearing-actuals-edit-summary.component.html [hunk 4]
```diff
@@ -1,204 +1,217 @@
...
+                  <div class="hearing-table-row">
+                    <div class="govuk-summary-list__row">
+                      <dt class="govuk-summary-list__key">
+                        <span class="govuk-heading-m govuk-!-margin-bottom-0">{{ 'Hearing timings' | rpxTranslate
+                          }}</span>
+                      </dt>
+                      <dd class="govuk-summary-list__value"></dd>
+                      <dd class="govuk-summary-list__actions">
+                        <a href="javascript:void(0)" class="govuk-link" [routerLink]="hearingActualAddEditUrl()"
+                          fragment="hearingDate">Change</a>
```
Comment (johnbenjamin-hmcts) [inline src/hearings/containers/hearing-actuals/hearing-actuals-edit-summary/hearing-actuals-edit-summary.component.html:110]: Translation for Change link.

File: src/hearings/containers/hearing-actuals/hearing-actuals-edit-summary/hearing-actuals-edit-summary.component.html [hunk 5]
```diff
@@ -1,204 +1,217 @@
...
+                  <br>
+                  <div class="hearing-table-row">
+                    <div class="govuk-summary-list__row">
+                      <dt class="govuk-summary-list__key">
+                        <span class="govuk-heading-m govuk-!-margin-bottom-0">{{ 'Participants' | rpxTranslate }}</span>
+                      </dt>
+                      <dd class="govuk-summary-list__value"></dd>
+                      <dd class="govuk-summary-list__actions">
+                        <a href="javascript:void(0)" class="govuk-link" [routerLink]="hearingActualAddEditUrl()"
+                          fragment="hearingDate">Change</a>
```
Comment (johnbenjamin-hmcts) [inline src/hearings/containers/hearing-actuals/hearing-actuals-edit-summary/hearing-actuals-edit-summary.component.html:145]: Translation for Change link.

File: src/hearings/containers/request-hearing/hearing-attendance/hearing-attendance.component.ts [hunk 6]
```diff
@@ -115,7 +115,11 @@ export class HearingAttendanceComponent extends RequestHearingPageFlow implement
 
   public getIndividualParties(): PartyDetailsModel[] {
     const individualParties: PartyDetailsModel[] = [];
+    const onPeperHearing = this.attendanceFormGroup.controls.paperHearing.value === RadioOptions.YES;
```
Comment (johnbenjamin-hmcts) [inline src/hearings/containers/request-hearing/hearing-attendance/hearing-attendance.component.ts:118]: Typo in onP**e**perHearing.

**Review by johnbenjamin-hmcts — CHANGES_REQUESTED** [review]
There are few changes requested, and one additional comment about moving the following block of code into a separate file and reference it in the .spec files

```
@Pipe({ name: 'rpxTranslate' })
class RpxTranslateMockPipe implements PipeTransform {
  public transform(value: string): string {
    return value;
  }
}
```

File: src/hearings/components/hearing-amend-warning-msg/hearing-amend-warning-msg.component.html [hunk 7]
```diff
@@ -1,7 +1,7 @@
 <div *ngIf="isViewEditMode" class="govuk-warning-text">
   <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
   <strong class="govuk-warning-text__text">
-    <span class="govuk-warning-text__assistive">Warning</span>
+    <span class="govuk-warning-text__assistive">{{ 'Warning' | rpxTranslate }}</span>
     {{ warningMsg }}
```
[the remark this answers is not in the document]
Comment (DavidJayakumar) [reply src/hearings/components/hearing-amend-warning-msg/hearing-amend-warning-msg.component.html]: Added

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by udaydenduluri33 — APPROVED** [review]

**Pull request merged**